index.ts

```diff
@@ -20,38 +20,68 @@ export interface EntryInfo {
   basename: string;
 }
 export type PathOrDirent = Dirent | Path;
-export type Tester = (path: EntryInfo) => boolean;
+export type Tester = (entryInfo: EntryInfo) => boolean;
 export type Predicate = string[] | string | Tester;
-
-function defaultOptions() {
-  return {
-    root: '.',
-    fileFilter: (_path: EntryInfo) => true,
-    directoryFilter: (_path: EntryInfo) => true,
-    type: FILE_TYPE,
-    lstat: false,
-    depth: 2147483648,
-    alwaysStat: false,
-    highWaterMark: 4096,
-  };
+export enum EntryTypes {
```

Comment:
use `const enum` https://ultimatecourses.com/blog/const-enums-typescript

Reply:
as you wish